src/zzz_od/win_exe/runtime_launcher.py

```
@@ -0,0 +1,76 @@
+import importlib
+import os
+import sys
+
+from one_dragon.launcher.exe_launcher import ExeLauncher
+from one_dragon.version import __version__
+
+# src/ 目录检查 — 仅 PyInstaller 打包环境下检查
+if getattr(sys, 'frozen', False):
+    _SRC_DIR = os.path.join(os.path.dirname(sys.executable), "src")
+    if not os.path.isdir(_SRC_DIR):
+        print(f"错误: 缺少 src 目录 ({_SRC_DIR})，请重新解压完整的 RuntimeLauncher 压缩包。")
```

Ruff (0.15.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)